test/controllers/admin_controller_test.rb

```diff
@@ -7,23 +7,13 @@ class AdminControllerTest < ActionController::TestCase
     assert_response :success
   end
 
-  test 'should get internal' do
-    get :internal
+  test 'should get health' do
+    get :health
     assert_response :success
   end
 
-  test 'should get users' do
-    get :users
-    assert_response :success
-  end
-
-  test 'should get user' do
-    get :user
-    assert_response :success
-  end
```

Comment:
There are no actions with these names, and I couldn't find any trace of them in the history of the `AdminController` or `config/routes.rb`.